Core/src/main/java/com/github/intellectualsites/plotsquared/plot/events/Result.java

```diff
@@ -0,0 +1,67 @@
+package com.github.intellectualsites.plotsquared.plot.events;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Enum for {@link CancellablePlotEvent}.
+ * <p>
+ * DENY: do not allow the event to happen
+ * ALLOW: allow the event to continue as normal, subject to standard checks
+ * FORCE: force the event to occur, even if normal checks would deny.
+ * WARNING: this may have unintended consequences! Make sure you study the appropriate code before using!
+ */
+public enum Result {
+
+    DENY(0), ACCEPT(1), FORCE(2);
+
+    private static Map<Integer, Result> map = new HashMap<>();
+
+    static {
+        for (Result eventResult : Result.values()) {
+            map.put(eventResult.value, eventResult);
+        }
+    }
+
+    private String reason = "Cancelled by external plugin.";
```

Comment:
This may or may not be replaced with a reason set by another plugin, but if it is set to "A" by plugin a and plugin b doesn't change it, the user will get a weird reason string. (The same applies for multiple reasons set by different threads)

Reply:
That's true. Perhaps remove the ability to set it all together? It's more of a nicety than a necessity,